storage/cloudspanner/admin.go

```diff
@@ -622,14 +623,8 @@ func (t *adminTX) UndeleteTree(ctx context.Context, treeID int64) (*trillian.Tre
 }
 
 func toTrillianTree(info *spannerpb.TreeInfo) (*trillian.Tree, error) {
-	createdPB, err := ptypes.TimestampProto(time.Unix(0, info.CreateTimeNanos))
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "failed to convert creation time: %v", err)
-	}
-	updatedPB, err := ptypes.TimestampProto(time.Unix(0, info.UpdateTimeNanos))
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "failed to convert creation time: %v", err)
-	}
+	createdPB := timestamppb.New(time.Unix(0, info.CreateTimeNanos))
+	updatedPB := timestamppb.New(time.Unix(0, info.UpdateTimeNanos))
```

Comment:
Might want CheckValid on these to preserve behaviour though not sure if these times are used in calculations anywhere.

Reply:
Added the checks back in.